python/afdko/autohint.py

```diff
@@ -1303,11 +1305,15 @@ def hintFile(options):
 					logMsg("No new hints. All glyphs were already hinted.")
 			else:
 				logMsg("No glyphs were hinted.")
+
 	if options.usePlistFile and (anyGlyphChanged or pListChanged):
 		#  save font plist file.
-		fontPlist.write(fontPlistFilePath)
+		with open(fontPlistFilePath, 'wb') as f:
+			plistlib.writePlist(fontPlist, f)
```

Comment:
use `dump()` instead?

Reply:
Eh. If we were planning to keep `autohint.py` current I guess that would be the right thing to do (actually, the right thing would be to use `fontTools.misc.plistlib`)

Reply:
only suggested it because you used `plistlib.load()` upstream

Reply:
Indeed. I'll change it to `dump()` in next round of changes.